### PR TITLE
Update to use bnd directives for custom jakarta bundle settings

### PR DESCRIPTION
**Pull request by jhanders34**
- Update to use the properties in tasks.gradle for setting bundle symbolic link (id), name and description.
- Update to use property already defined in tasks.gradle to have the name of the jar file match the name of the symbolic name.

### CODE REVIEW

**Comment by jhanders34** [comment]
#build

File: dev/com.ibm.ws.security.sso/bnd.bnd [hunk 1]
```diff
@@ -19,11 +19,10 @@ Bundle-Description: WAS Security SSO 1.0, version=${bVersion}
...
 jakartaeeMe: true
-#jakartaee.transform.jar.name=io.openliberty.security.sso.internal.jar
-#JakartaEE-Bundle-Name: Security Single Sign-On Implementation
-#JakartaEE-Bundle-Symbolic-Name: io.openliberty.security.sso.internal
-#JakartaEE-Bundle-Description: Security Single Sign-On Implementation, version=2.0
-#JakartaEE-Bundle-Version: 2.0
+jakartaFinalJarName: io.openliberty.security.sso.internal.jar
+jakartaFinalBundleName: Security Single Sign-On
+jakartaFinalBundleId: io.openliberty.security.sso.internal
+jakartaFinalBundleDescription: Security Single Sign-on
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.sso/bnd.bnd]:
```suggestion
jakartaFinalBundleDescription: Security Single Sign-on; Jakarta Enabled
```

File: dev/com.ibm.ws.security.authentication.filter/bnd.bnd [hunk 2]
```diff
@@ -19,11 +19,10 @@ Bundle-Description: Security Authentication filter, version=${bVersion}
...
 jakartaeeMe: true
-#jakartaee.transform.jar.name=io.openliberty.security.authentication.internal.filter.jar
-#JakartaEE-Bundle-Name: Security Authentication filter
-#JakartaEE-Bundle-Symbolic-Name: io.openliberty.security.authentication.internal.filter
-#JakartaEE-Bundle-Description: Security Authentication filter
-#JakartaEE-Bundle-Version: 2.0
+jakartaFinalJarName: io.openliberty.security.authentication.internal.filter.jar
+jakartaFinalBundleName: Security Authentication filter
+jakartaFinalBundleId: io.openliberty.security.authentication.internal.filter
+jakartaFinalBundleDescription: Security Authentication filter
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.authentication.filter/bnd.bnd]:
```suggestion
jakartaFinalBundleDescription: Security Authentication filter; Jakarta Enabled
```

File: dev/com.ibm.websphere.appserver.api.jacc/bnd.bnd [hunk 3]
```diff
@@ -19,11 +19,10 @@ Bundle-SymbolicName: com.ibm.websphere.appserver.api.jacc
 # Produce a JakartaEE compliant JAR.
 #
 jakartaeeMe: true
-#jakartaee.transform.jar.name=io.openliberty.jacc.2.0.jar
-#JakartaEE-Bundle-Name: JACC Service Provider Interface
-#JakartaEE-Bundle-Symbolic-Name: io.openliberty.jacc.2.0
-#JakartaEE-Bundle-Description: JACC Service Provider Interface, version 2.0
-#JakartaEE-Bundle-Version: 2.0
+jakartaFinalJarName: io.openliberty.jacc.2.0.jar
+jakartaFinalBundleName: JACC Service Provider Interface
```
Comment (jvanhill) [inline dev/com.ibm.websphere.appserver.api.jacc/bnd.bnd:23]: For JASPIC / com.ibm.websphere.appserver.spi.jaspic bundle we spell out the acronym, do we want to do the same here, or use JASPIC in that bundle as well?

**Review by jvanhill — APPROVED** [review]
Just a few nits I found. They can be ignored if desired. For the most part, I just wanted to be able to differentiate the bundles when there was a JavaEE and JakartaEE equivalent. Either by the spec name / version, or a description in the bundle itself.

File: dev/com.ibm.websphere.appserver.api.jacc/bnd.bnd (hunk 3 above)
Comment (jhanders34) [reply dev/com.ibm.websphere.appserver.api.jacc/bnd.bnd:23]: Well it isn't JASPIC or JACC any longer with Jakarta.  It is now Jakarta Authentication and Jakarta Authorization respectively.

File: dev/com.ibm.ws.security.authentication.filter/bnd.bnd (hunk 2 above)
Comment (jhanders34) [reply dev/com.ibm.ws.security.authentication.filter/bnd.bnd]: Updated

File: dev/com.ibm.ws.security.sso/bnd.bnd (hunk 1 above)
Comment (jhanders34) [reply dev/com.ibm.ws.security.sso/bnd.bnd]: Updated

**Comment by jhanders34** [comment]
#libby #build

**Comment by jhanders34** [comment]
#libby #build

**Pull request merged**